src/spaced/spaced/game/world.cpp

```diff
@@ -0,0 +1,104 @@
+#include <bave/imgui/im_text.hpp>
+#include <spaced/game/enemy_factory_builder.hpp>
+#include <spaced/game/world.hpp>
+
+#include <spaced/game/controllers/auto_controller.hpp>
+#include <spaced/game/controllers/player_controller.hpp>
+
+namespace spaced {
+using bave::FixedString;
+using bave::NotNull;
+using bave::Seconds;
+using bave::Shader;
+
+namespace {
```

Comment:
why the unnamed namespace here? are these just `World` helper functions?

Reply:
Yeah they are helper functions that are only used in this cpp file, so putting them in an unnamed namespace gives them internal linkage (visible only to this cpp file) and the compiler a better opportunity to optimize them.